meteor/server/api/playout/playout.ts

```diff
@@ -1483,7 +1486,13 @@ export namespace ServerPlayoutAPI {
 			}
 
 			waitForPromise(cache.saveAllToDatabase())
+
+			takeAfterExecute = context.takeAfterExecute
 		})
+
+		if (takeAfterExecute) {
+			Meteor.call(UserActionAPIMethods.take, 'e', rundownPlaylistId)
```

Comment:
Since this is called from the server-side, there's no reason why this should use the UserActionAPI, the 'e' string as userAction description will only be confusing (as no such action is actually made by the user). I think this just be a straight call to `ServerPlayoutAPI.rundownTake()`.